packages/drafts-realm/chess.gts

```diff
@@ -0,0 +1,466 @@
+import {
+  Component as BoxelComponent,
+  CardDef,
+} from 'https://cardstack.com/base/card-api';
+import Component from '@glimmer/component';
+import StringCard from 'https://cardstack.com/base/string';
+import { contains, field } from 'https://cardstack.com/base/card-api';
+
+import lodash from 'https://cdn.jsdelivr.net/npm/lodash@4.17.21/+esm';
+import {
+  Chessboard,
+  FEN,
+} from 'https://cdn.jsdelivr.net/npm/cm-chessboard@8.7.3/+esm';
+
+import Modifier from 'ember-modifier';
+
+type ChessboardModifierSignature = {
+  Args: {
+    Named: {
+      fen?: string;
+    };
+  };
+  Element: HTMLElement;
+};
+
+class ChessboardModifier extends Modifier<ChessboardModifierSignature> {
+  chessboard: any;
+  modify(
+    element: HTMLElement,
+    _positional: [],
+    { fen }: ChessboardModifierSignature['Args']['Named'],
+  ) {
+    debugger;
+    if (this.chessboard == undefined) {
+      this.chessboard = new Chessboard(element, {
+        position: fen ?? FEN.start,
+        style: {
+          pieces: {
+            file: 'https://cdn.jsdelivr.net/npm/cm-chessboard@8.7.3/assets/pieces/standard.svg',
```

Comment:
it was a good a library enabled this so I didn't have to download the assets somehow

Reply:
nice, although I was able to delete `style` altogether and it still rendered so maybe it has a default sprite set?

Reply:
@backspace perhaps this is what you meant. You can just use assetsUrl directly. Which you can!

Reply:
ya my comment about deleting `style` was incorrect as we saw in the meeting today, I must have been looking at a cached version. I’m still murky on what it takes for a card to be properly rerendered when its module has changed.

Reply:
FYI. I dont think this cdn of assets is signficant to the loader we use at all. Its just simply a url that gets link to a href. 

Look here https://github.com/shaack/cm-chessboard/blob/7918204d91aa6c1916f1ed1ca804524bca43b718/src/view/ChessboardView.js#L462-L467 

https://github.com/shaack/cm-chessboard/blob/7918204d91aa6c1916f1ed1ca804524bca43b718/src/view/ChessboardView.js#L44-L46

https://github.com/shaack/cm-chessboard/blob/7918204d91aa6c1916f1ed1ca804524bca43b718/src/view/ChessboardView.js#L98-L102